src/frameworks/compose/index.ts

```diff
@@ -1,31 +1,36 @@
 import { AppBundle } from "./interfaces";
 import { getDriver, Mode } from "./driver";
 import { discover } from "./discover";
+import { FrameworkSpec, FileSystem } from "./discover/types";
 
 /**
  * Run composer in the specified execution context.
  */
-export function compose(mode: Mode): AppBundle {
+export async function compose(
```

Comment:
please also update `src/commands/internaltesting-frameworks-compose.ts` call to `compose` to await on the result.

Reply:
Noticed that unit tests are failing due to this.